### PR TITLE
Added support for loadCSS and async javascripts

### PR DESCRIPTION
**Pull request by imathis**
This should fix octopress/asset-pipeline#15

### CODE REVIEW

**Comment by imathis** [comment]
Basically you'll be able to add this to your `_config.yml` and it will automatically load your CSS asynchronously, but only if it is building the combined file (for production).

```
asset_pipeline:
  async_css: true
```

This also sets async javascript to true for combined (production) script tags. Since the order is already defined by the plugins (or author in the case of octopress-asset-pipeline), this accomplishes the same as `defer` but loads the js in asynchronously.

File: test/standard/_expected/async_css/asset_tag.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<script>/*!
```
Comment (imathis) [inline test/standard/_expected/async_css/asset_tag.html]: @kogakure Here you can see the output that you'll get if you set your site configuration to:

```
asset_pipeline:
  async_css: true
```

**Comment by imathis** [comment]
[Here is what it looks like](https://github.com/octopress/ink/pull/56/files#diff-0d386837cb358a4902cdaf38e90c12a0R1) when you generate with `async_css: true`.

**Comment by kogakure** [comment]
Nice! All I thought of was to have a path and you include loadCSS and all things needed by default :open_mouth: Thank you for this cool solution! :+1:

**Comment by imathis** [comment]
I started out looking at what it would take to make it work with paths, but the implementation using paths ended up being really inelegant. So I decided that I might as well make it work the way I'd want to use it. Thanks for drawing my attention to this. :)

**Pull request merged**